tests/Unit/Elliptic/DiscontinuousGalerkin/SubdomainOperator/Test_SubdomainOperator.cpp

```diff
@@ -200,23 +200,25 @@ struct RandomBackground : elliptic::analytic_data::Background {
   WRAPPED_PUPable_decl_template(RandomBackground);  // NOLINT
 #pragma GCC diagnostic pop
 
+  // NOLINTBEGIN(readability-convert-member-functions-to-static)
   template <typename... RequestedTags>
-  static tuples::TaggedTuple<RequestedTags...> variables(
+  tuples::TaggedTuple<RequestedTags...> variables(  // NOLINT
```

Comment:
Do you need to NOLINT twice?

Reply:
"Only" until we upgrade to clang-tidy 14, which adds support for `NOLINT{BEGIN,END}` :)